cogs/quotes.py

```diff
@@ -61,6 +61,29 @@ async def list(self, ctx, search=""):
         if len(text) > 0:
             await ctx.send("```{}```".format(text))
 
+    @quote.command()
+    @commands.has_permissions(administrator=True)
+    async def delete(self, ctx, search):
+        """Removes a quote"""
+
+        search = "%" + search + "%"
+
+        with self.bot.db.get(ctx.guild.id) as db:
+            results = db.execute("SELECT content FROM quotes WHERE LOWER(content) LIKE ? ORDER BY content",
+                                 (search,)).fetchall()
+
+        if len(results) > 1:
+            await ctx.send("Only one quote at a time can be removed, to prevent admin-abuse.")
+            return
+        if len(results) == 0:
+            await ctx.send("No quotes could be found. What does not exist cant be deleted. :pengsad:")
```

Comment:
Since this bot is also in use by other servers, please refrain from using custom emojis.

Reply:
like in johannes.py?

Reply:
johannes.py is more of an Easter Egg. It's planned to be replaced by a programmable autoreact module anyways (to avoid everything being global), at least I tend to tell myself that.